cmd/traffic/cmd/manager/internal/cluster/info.go

```diff
@@ -213,6 +213,23 @@ func NewInfo(ctx context.Context) Info {
 		dlog.Warnf(ctx, "Unable to get manager pod ip; env var says %s", env.PodIP)
 	}
 
+	oi.DnsConfig = &rpc.DNSConfig{}
+	oi.DnsConfig.AlsoProxySubnets, err = env.GetAlsoProxySubnets()
+	if err != nil {
+		dlog.Warnf(ctx, "AlsoProxySubnets not parsed: %v", err)
+	}
+	for _, subnet := range oi.DnsConfig.AlsoProxySubnets {
+		dlog.Infof(ctx, "Using AlsoProxySubnet: %+v", subnet)
+	}
+
+	oi.DnsConfig.NeverProxySubnets, err = env.GetNeverProxySubnets()
+	if err != nil {
+		dlog.Warnf(ctx, "NeverProxySubnets not parsed: %v", err)
```

Comment:
Why is this a warning? Looks like an error to me.

Reply:
yeah i agree